templates/operations.ts.hbs

```diff
@@ -121,7 +121,8 @@
         {{/if}}
       }>,
       rawResponse?: boolean,
```

Comment:
Make sure you add a deprecation notice to `rawResponse`. I know that `@deprecated` exists as a doc tag, but I don't know what scopes it can be applied to (i.e. can it be used for params or just for methods?).